jason-lab/routes/dead.js

```diff
@@ -0,0 +1,171 @@
+'use strict';
+
+const storage = require('../lib/storage.js');
+const DeadCharacter = require('../model/simple-resource.js');
+const Router = require('../lib/router.js');
+const router = new Router();
```

Comment:
☠️ this to avoid confusion with the `router` parameter that's passed to `allRoutes`.